source/MainMenuState.hx

```diff
@@ -68,10 +70,12 @@ class MainMenuState extends MusicBeatState
 
 		persistentUpdate = persistentDraw = true;
 
+		FlxG.mouse.visible = true;
+
 		var yScroll:Float = Math.max(0.25 - (0.05 * (optionShit.length - 4)), 0.1);
 		var bg:FlxSprite = new FlxSprite(-80).loadGraphic(Paths.image('menuBG'));
 		bg.scrollFactor.set(0, yScroll);
-		bg.setGraphicSize(Std.int(bg.width * 1.175*scaleRatio ));
+		bg.setGraphicSize(Std.int(bg.width * 1.175));
```

Comment:
Redundant change? The mouse doesn't need to be visible.

I know that `scaleRatio` needs an "overhaul" due to compilation targets, but not here.